docs/mission-makers/olsen-gear.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Using Olsen Gear Scripts
+nav_order: 2
+parent: Modules
+---
+
+# Olsen Gear Scripts via the UO Framework
+
+GearScript.sqf
+
+```sqf
+#define random(MIN, MAX) \
+([MIN, MAX] call UO_FW_FNC_RandomRange)
+_temp = "";
+#define SET_GROUP(groupName) _temp = #groupName + package;\
+call compile format ['%1 = {
+#define END_GROUP };', _temp]
+#define ADD_GROUP(groupName) call call compile format ["%1", #groupName + package]
+private ["_unit", "_type", "_groupId"];
+
+_unit = _this select 0;
+_type = _this select 1;
+
+if (!local _unit) exitWith {};
+
+_unit setVariable ["BIS_enableRandomization", false];
+_unit setVariable ["UO_FW_Loadout", _type, true];
+
+FNC_AddItem = {([_unit, _type] + _this) call UO_FW_FNC_AddItemOrg;};
+FNC_AddItemRandom = {([[_unit, _type]] + [_this]) call UO_FW_FNC_AddItemRandomOrg;};
+_unit call UO_FW_FNC_RemoveAllGear;
+```
+
+If you're porting gear scripts from the olsen framework you need to include this at the top of your GearScript.sqf file.
```

Comment:
I would put this sentence above the code and phrase it "... you need to include following code at the top ..."